ansible_collections/arista/avd/plugins/modules/inventory_to_container.py

```diff
@@ -433,7 +438,10 @@ def main():
                 inventory_content = yaml.safe_load(stream)
             except yaml.YAMLError as exc:
                 module.debug(exc)
-        result['CVP_TOPOLOGY'] = get_containers(inventory_content=inventory_content,
+                raise AnsibleValidationError(
+                    f"Failed to parse inventory file, original exception:{exc}"
```

Comment:
Do you need to print out the exception? Normally, when raising with `from` it will already print out both exceptions.

Reply:
I did not want to remove the `module.debug` log that was already there in case it was needed by someone but happy to remove it indeed

Reply:
Hmm ok - read too fast - I have now removed the `{exc}` in the error message..